custom-plugins/fictive_studios/admin/builder-2d/partials/builder-2d.php

```diff
@@ -0,0 +1,6 @@
+<div class="wrap">
+    <h2>
+        <?php esc_html_e('Create Template', 'admin-table-tut'); ?>
+        <a id="editor2d" class="button-secondary" href="<?php echo $link_url; ?>" target="_blank">Open Template Editor</a>
```

Comment:
Link not working.

Reply:
this will be done after the base table is ready